### PR TITLE
revamped contact-us section

### LINKED ISSUE #509: Improve Contact Section
[Reference](https://jacekjeznach.com/contact/)

### Suggestion
Use Leaflet.JS, OpenMap API ( There are some cool walkthroughs on youtube if you wish to refer)

### PR DESCRIPTION
**Pull request by ashuvssut**
closes #509 

- revamped contact-us form
- added spl hover effect on SEND button😍
- made a tooltip in map
- map's marker is on chennai, India
- map's height set to vary for best UX

Couldnt find you email Id anywhere so I added the link to your portfolio in the map tooltip instead

**EDIT**: Added an extra feature: RE-CENTER button on map😁

-----------------------
_--I have participated in DWoC SWoC and MWoC_

### CODE REVIEW

**Comment by ashuvssut** [comment]
took lot of tym😅 Just improving the UI and making it responsive took too much time. Hope u ll like it😁

**Comment by smaranjitghose** [comment]
@ashuvssut  Please remove my name and email from the map 😅. All of us deserve credit for the project so it won't be fair if my name alone is there on the map.

Also, try to modify the map as per the reference I gave in the issue

**Comment by smaranjitghose** [comment]
Also, I think the Contact Us Section is overkill for the Home Page, could you please shift it to a standalone page and update the navbar (with routing)

**Comment by ashuvssut** [comment]
okay I will make a new page for contact us page. for that I have to update the navbar. got it

**Comment by ashuvssut** [comment]
>Also, try to modify the map as per the reference I gave in the issue

Do you want a dark theme map??

**Comment by smaranjitghose** [comment]
> > Also, try to modify the map as per the reference I gave in the issue
> 
> Do you want a dark theme map?
> I will implement dark theme in this same PR only (if you want a dark theme map now)

Yes I meant the dark theme map (with the sea blue-ish color)

**Comment by smaranjitghose** [comment]
> @smaranjitghose
> 
> okay I will make a follow up PR for a new page for contact us section. for that I have to update the navbar also. got it!
> 
> **And one question**... there is a marker in that map...that also shows you name and address when clicked
> 
> Are you asking to remove the Tooltip and that market popup both??
> Do you want anything else in those tooltip and popup's content?

Just set it to 

Doc2Pen
contact@doc2pen.org for now

File: src/pages/Home/components/Map/Map.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <br />
+        <br />
+        <a href="mailto:contact@doc2pen.org">contact@doc2pen.org</a>
+      </div>
+      <TileLayer
+        attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+        url="https://{s}.tile.thunderforest.com/transport-dark/{z}/{x}/{y}.png"
+        // url="https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png"
+        // url="https://stamen-tiles-{s}.a.ssl.fastly.net/toner/{z}/{x}/{y}{r}.png"
+        //url='https://{s}.basemaps.cartocdn.com/dark_all/{z}/{x}/{y}{r}.png
```
Comment (ashuvssut) [inline src/pages/Home/components/Map/Map.jsx:35]: here
```js
url="https://{s}.tile.thunderforest.com/transport-dark/{z}/{x}/{y}.png"

// url="https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png"
// url="https://stamen-tiles-{s}.a.ssl.fastly.net/toner/{z}/{x}/{y}{r}.png"
//url='https://{s}.basemaps.cartocdn.com/dark_all/{z}/{x}/{y}{r}.png
```

I was able to found only 4 dark themes that do not require a api key

the current applied one is the best among all four

I search through many sites to get the dark theme url but most of them required an api key

**Comment by ashuvssut** [comment]
In my next PR 
- i will make a brand new page for the contact us section. 
- I will update navbar and routing. 
- I will style the map components

**Comment by anushbhatia** [comment]
Sure now you can work further on this

**Pull request merged**